### DISCUSSION TITLE
SSH command timed out

### DISCUSSION THREAD

**Discussion by pollynesterovich** [apache/airflow#29697] [General]
### Official Helm Chart version

1.8.0 (latest released)

### Apache Airflow version

2.5.1

### Kubernetes Version

v1.21.13

### What happened

When starting the dag we get an error:
`airflow.exceptions.AirflowException: SSH command timed out`

### How to reproduce

Run dag with configuration:
`SSHOperator(
                task_id=team_app.replace(":", "_"),
                command=cmd_load,
                ssh_conn_id="",
                do_xcom_push=True
            )`

### Anything else

Please add this [mr](https://github.com/apache/airflow/pull/29347) to the [helm chart](https://github.com/apache/airflow/tree/main/chart) of the latest version 1.8.0 so it affects the work of dags.

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by potiuk** [comment]
Your SSH conection must have timed out. This description doe snot indicate what is the expected behaviour nor describe your deployment. I have no idea what your comment about MR is (did you add it? Should we add it? Should it not be added? No idea.

If you have a problem with ssh timeout then it is likely your ssh command takes long. No idea. If you need help with troubleshooting, you can continue the discussion here, but please do not create an issue for things that are troubleshooting your case, only when you clearly see there is an issue in Airlfow and have reproducible case.
